common/common.cpp

```diff
@@ -631,6 +631,7 @@ bool gpt_params_parse(int argc, char ** argv, gpt_params & params) {
         process_escapes(params.prompt);
         process_escapes(params.input_prefix);
         process_escapes(params.input_suffix);
+        process_escapes(sparams.cfg_negative_prompt);
```

Comment:
Keep only this line from this PR and we can merge.
All other changes are not necessary